src/biblenotify/ui/BibleNotify/Views/HomeView.qml

```diff
@@ -9,38 +9,65 @@ Item {
     id: root
 
     property var setTimeView
+    property string verseText
+    property string verseReference
 
     signal sendNotification
 
     ColumnLayout {
         anchors.fill: parent
 
-        Image {
+        ColumnLayout {
             Layout.alignment: Qt.AlignHCenter
-            source: "qrc:/illustration-with_text.svg"
-            antialiasing: true
+
+            BNLabel {
+                Layout.alignment: Qt.AlignHCenter
+                Layout.maximumWidth: 600
+                horizontalAlignment: Text.AlignHCenter
+                font.pixelSize: 30
+                color: "#24292f"
+                text: qsTr(verseText)
```

Comment:
No point being able to translate this text. This line should read `text: root.verse[0]` with these and a couple other changes.